scripts/init-all.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+if [ ! -f docker-compose.yml ]; then
+    echo "You are not inside faf-stack! The working directory must be the root of faf-stack."
+    exit 1
+fi
+
+[ ! -d config ] && ln -s config.template config
+[ ! -f .env ] && ln -s .env.template .env
```

Comment:
This should also print that a symbolic link was created. There is a tiny risk here, that you commit back changes you just did for local testing.

Reply:
Thanks, done.